### PR TITLE
Fixes #14784: Add a policy to manage relay server

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/14784

### CODE REVIEW

File: techniques/system/common/1.0/failsafe.st [hunk 1]
```diff
@@ -44,8 +44,7 @@ bundle common g
 
     any::
       "uuid"                     string => readfile("${g.uuid_file}", 60);
-      "excludedreps"             slist  => { "\.X11", ".*kde.*", "\.svn", "perl" };
```
Comment (amousset) [inline techniques/system/common/1.0/failsafe.st]: No idea what is was there for but it should be necessary...

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/update.cf [hunk 2]
```diff
@@ -166,7 +167,6 @@ bundle agent update_action
     initial_promises.root_server::
       "${g.rudder_ncf}/common"
         copy_from         => copy_digest_without_perms("${g.rudder_ncf_origin_common}"),
-        depth_search      => recurse_ignore("inf", @{g.excludedreps}),
```
Comment (ncharles) [inline techniques/system/common/1.0/update.cf]: removing it prevents update of ncf

**Pull request merged**